File: benchmarks/python/10_primes_count.py
def is_prime(n):
    if n < 2: return False
    if n < 4: return True
    if n % 2 == 0: return False
    d = 3
    while d * d <= n:
        if n % d == 0: return False
        d += 2
    return True

def count_primes(limit):
    count = 0
    for n in range(2, limit):
        if is_prime(n):
            count += 1
    return count
print(count_primes(10000000))

File: benchmarks/node/10_primes_count.mjs
function is_prime(n) {
    if (n < 2) return false;
    if (n < 4) return true;
    if (n % 2 === 0) return false;
    for (let d = 3; d * d <= n; d += 2) if (n % d === 0) return false;
    return true;
}
function count_primes(limit) {
    let count = 0;
    for (let n = 2; n < limit; n++) if (is_prime(n)) count++;
    return count;
}
console.log(count_primes(10000000));
